core/PaginatedList.php

```diff
@@ -457,4 +474,30 @@ public function getRequest() {
 		return $this->request;
 	}
 
+	/**
+	 * Remove the first page pagination get var to stop duplicate page issues
+	 * / and /?start=0 return the same page
+	 *
+	 * @param  string $link
+	 * @return string
+	 */
+	private function removeFirstPagePaginationParam($link)
```

Comment:
And if anyone is using multiple `?` to join all their querystrings, I hope their site breaks....